.github/workflows/linter.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
```

Comment:
[OPTIONAL]

- I noticed that there are two files in the `.github/workflows` folder. Kindly remove the `linter.yml` file as, according to the [linters configuration](https://github.com/microverseinc/linters-config) document, only the `linters.yml` file is needed in the folder and you already have that. 😀